css/css-contain/content-visibility/spacer-with-popover-top-layer-ref.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<meta charset="utf8">
+<title>CSS Content Visibility: popover shows under c-v auto (ref)</title>
+
+<style>
+.box { width: 100px; height: 100px; border: 1px solid black; }
+.spacer { width: 10px; height: 3000px; background: lightblue; }
+</style>
+
+<div class=spacer></div>
+<div id=container class=box>
+content
```

Comment:
Should we remove `content` from the ref? Since it's offscreen, it's not relevant to the user, therefore is skipped right?

Reply:
I rather keep this as-is since it is based on an original test.